packages/block-library/src/post-featured-image/edit.js

```diff
@@ -196,6 +196,16 @@ export default function PostFeaturedImageEdit( {
 		}
 	};
 
+	// On reset image
+	const onResetImage = () => {
+		setAttributes( {
+			isLink: false,
+			linkTarget: '_self',
+			rel: '',
+		} );
```

Comment:
```suggestion
		setAttributes( {
			isLink: false,
			linkTarget: '_self',
			rel: '',
			sizeSlug: undefined,
		} );
```

We should reset the `sizeSlug` attribute. Otherwise, the redundant comment remains after resetting a media:

https://github.com/user-attachments/assets/920c0111-db52-42e0-926e-7e45d5ca2da0

Reply:
added it to handler